src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -12,21 +12,22 @@
 
 def import_lazily():
     "Makes annotations / typing available lazily"
-    global NWBFile, ElectricalSeries, NWBHDF5IO
+    global NWBFile, ElectricalSeries, Units, NWBHDF5IO
     from pynwb import NWBFile
     from pynwb.ecephys import ElectricalSeries
+    from pynwb.misc import Units
     from pynwb import NWBHDF5IO
 
 
-def retrieve_electrical_series(nwbfile: NWBFile, electrical_series_name: Optional[str] = None) -> ElectricalSeries:
+def retrieve_electrical_series(nwbfile: NWBFile, electrical_series_path: Optional[str] = None) -> ElectricalSeries:
```

Comment:
I like the convention that you have on the other functions of using _pynwb or _backend to divided the modes. I also think that we should probably make all of these methods private

Reply:
might be useful externally as well IMO

Reply:
They might but I don't want to mantain or keep the promise of not changing their signature so if people use them they should be aware that we might change them. Hence private.